app/konnect/gateway-manager/data-plane-nodes/index.md

```diff
@@ -4,14 +4,15 @@ content_type: explanation
 disable_image_expand: true
 ---
 
-A runtime instance is a single self-managed instance of {{site.base_gateway}} that functions as a data plane. In {{site.konnect_saas}}, runtime instances are part of [runtime groups](/konnect/runtime-manager/runtime-groups) and can be used to serve traffic to the runtime group. Runtime groups manage and store configuration in {{site.konnect_saas}}, and runtime instances are configured according to the configuration distributed by the runtime group.
+A data plane node is a single self-managed instance of {{site.base_gateway}} that acts as a proxy and serves traffic.
+In {{site.konnect_saas}}, data planes are managed by [control planes](/konnect/gateway-manager/control-plane-groups/). Control planes manage and store configurations in {{site.konnect_saas}}, and data plane nodes are configured according to the configuration distributed by the control plane.
```

Comment:
In this paragraph, we jump from talking about data plane nodes in the previous sentence to talking about data planes, but a data plane isn't explained in relation to data plane nodes. I think data planes have one (or more?) nodes but I don't understand how the two interact, so selfishly I want more information here about how the two work together/relate.

Reply:
In this situation, the terms data planes and data plane nodes are being used interchangeably. 

Technically, the definitions are:

control plane: manages config for the data plane
data plane: a collection of nodes that process traffic. they accept config from the data plane and implement it on all the traffic that goes through.
This is more of an abstract term.
data plane node: one Kong Gateway instance.

We really should be saying "data plane node" if we just mean one node. "data plane" refers to the whole. "Data planes" no longer makes sense if we're introducing the term "data plane nodes".